src/app/tasks/create-task/create-task.component.ts

```diff
@@ -25,20 +30,35 @@ export class CreateTaskComponent {
     description: new FormControl(),
     type: new FormControl(null, Validators.required),
     createdOn: new FormControl(new Date().toDateString()),
-    status: new FormControl('In Progress', Validators.required)
+    status: new FormControl(null, Validators.required)
   });
 
-  constructor(private taskService: TaskService) {}
+  constructor(private taskService: TaskService) {
+    // Map the task type enum to array of keys and values
+    Object.keys(TaskType).forEach(taskTypeKey => {
```

Comment:
same here instead of going through the keys you can just go through the entries of the enum and have both key and value of that key